pkg/kubectl/cmd/delete/delete.go

```diff
@@ -93,6 +93,7 @@ var (
 		kubectl delete pods --all`))
 )
 
+// DeleteOptions has the data required to perform the delete operation
 type DeleteOptions struct {
```

Comment:
You should remove this package from `golint-failure` file which would lead to lint error since this struct starts with `delete`.

Reply:
Thank you for your review. @draveness 

The reason I didn't remove the file from .golint-failures is because of this very lint error. As per the contributing guide for the cli and a discussion [here](https://github.com/kubernetes/kubernetes/pull/70682#discussion_r233642556), changing `delete.DeleteOptions` to `delete.Options` is not a suggested approach.